routing/road_access.cpp

```diff
@@ -1,25 +1,127 @@
 #include "routing/road_access.hpp"
 
+#include <algorithm>
 #include <sstream>
 
+using namespace std;
+
+namespace
+{
+string const kNo = "No";
+string const kPrivate = "Private";
+string const kDestination = "Destination";
+string const kYes = "Yes";
+string const kCount = "Count";
+string const kNames[] = {kNo, kPrivate, kDestination, kYes, kCount};
+
+// *NOTE* Order may be important for users (such as serializers).
```

Comment:
Probably it's worth adding that new types should be added at the end of the list?